### PR TITLE
Update naming of Ui.png file

### PR DESCRIPTION
**Pull request by matthewyip1511**
From the website, the `Ui.png` file is a special file that must showcase our app. Right now, the file points to the old Ui (the prototype that @yanqiyqh drew) while the new screenshot has a different name. This PR changes the name of the files to allocate the `Ui.png` to the screenshot.

However, this is also a good time to look at whether the screenshot is a good representation of our app. Right now it mostly shows the claims and insurance plans, but if anyone thinks there is more to include I can do so too. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [![codecov](https://codecov.io/gh/AY2425S1-CS2103-F12-1/tp/graph/badge.svg?token=4P6LHV4HJ0)](https://codecov.io/gh/AY2425S1-CS2103-F12-1/tp)
 
-![appScreenshot.png](docs/images/appScreenshot.png)
+![appScreenshot.png](docs/images/Ui.png)
```
Comment (yanqiyqh) [inline README.md:5]: Should we update `appScreenshot.png` to match the new `Ui.png` name?

**Review by yanqiyqh — COMMENTED** [review]
Just one small nit

File: README.md (hunk 1 above)
Comment (matthewyip1511) [reply README.md:5]: we should! thanks for picking this up :)

Comment (matthewyip1511) [reply README.md]: changed :)

**Review by RezwanAhmed123 — APPROVED** [review]
LGTM

**Pull request merged**